connectors/test_connectors.py

```diff
@@ -9,11 +9,11 @@ class ConnectorsTestCase(DataprocTestCase):
     COMPONENT = "connectors"
     INIT_ACTIONS = ['connectors/connectors.sh']
 
-    BQ_CONNECTOR_VERSION = "1.1.1"
-    BQ_CONNECTOR_URL = "gs://hadoop-lib/bigquery/bigquery-connector-hadoop2-1.1.1.jar"
+    BQ_CONNECTOR_VERSION = "1.2.0"
+    BQ_CONNECTOR_URL = "gs://hadoop-lib/bigquery/bigquery-connector-{}-1.2.0.jar"
 
-    GCS_CONNECTOR_VERSION = "2.1.1"
-    GCS_CONNECTOR_URL = "gs://hadoop-lib/gcs/gcs-connector-hadoop2-2.1.1.jar"
+    GCS_CONNECTOR_VERSION = "2.2.0"
+    GCS_CONNECTOR_URL = "gs://hadoop-lib/gcs/gcs-connector-{}-2.2.0.jar"
 
     SPARK_BQ_CONNECTOR_VERSION = "0.13.1-beta"
```

Comment:
Does this also need an update to 0.19.1 ?

Reply:
Done.